articles/tables.md

```diff
@@ -681,6 +692,35 @@ Clicking on any file will download it.
 {% endtab %}
 {% endtabs %}
 
+## Table Facets
+The faceted navigation on `Tables` can be used to help simplify your search without having to use SQL-like queries. To use table facets, navigate to a `Table` or a `File View`. For this example, we'll be using the [Synapse Table Demo](https://www.synapse.org/#!Synapse:syn3079449/tables/){:target="_blank"} found in the [Wondrous Research Example](https://www.synapse.org/#!Synapse:syn1901847/wiki/56044){:target="_blank"}.
```

Comment:
bit confusing to link to existing table that users can't see all schema buttons or modify it